gateway/gateway.go

```diff
@@ -542,6 +526,19 @@ func (gateway *HandleT) eventSchemaWebHandler(wrappedFunc func(http.ResponseWrit
 	}
 }
 
+func (gateway *HandleT) getPayloadFromRequest(r *http.Request) ([]byte, error) {
+	if r.Body != nil {
+		payload, err := ioutil.ReadAll(r.Body)
+		r.Body.Close()
+		return payload, err
+	}
+	return []byte{}, errors.New("RequestBodyNil")
+}
+
+func (gateway *HandleT) webImportHandler(w http.ResponseWriter, r *http.Request) {
+	gateway.webHandler(w, r, "import")
```

Comment:
Why forward this to webhandler and then do a check on reqType == 'import'. Instead just directly send to a different well defined function?